google-account-plugin/src/com/google/cloud/tools/intellij/login/GoogleLoginPrefs.java

```diff
@@ -82,17 +86,23 @@ public static void saveOAuthData(OAuthData credentials) {
    * an {@code OAuthData} object all of whose getters return {@code null} .
    */
   public static OAuthData loadOAuthData() {
-    Preferences prefs = getPrefs();
+    String refreshToken = null;
+    String storedEmail = null;
+    SortedSet<String> storedScopes = new TreeSet<String>();
 
-    String refreshToken = prefs.get(getCustomUserKey(OAUTH_DATA_REFRESH_TOKEN_KEY), null);
-    String storedEmail = prefs.get(getCustomUserKey(OAUTH_DATA_EMAIL_KEY), null);
-    String storedScopesString = prefs.get(getCustomUserKey(OAUTH_SCOPES_KEY), "");
+    if (Services.getLoginService().getActiveUser() != null) {
+      Preferences prefs = getPrefs();
 
-    // Use a set to ensure uniqueness.
-    SortedSet<String> storedScopes = new TreeSet<String>();
-    for (String scope : storedScopesString.split(DELIMITER)) {
-      storedScopes.add(scope);
+      refreshToken = prefs.get(getCustomUserKey(OAUTH_DATA_REFRESH_TOKEN_KEY), null);
```

Comment:
My guess is a bunch of this code is the way it is because the original credential store did not support multiple users and this tries to support both the schema with just one user, and the one with a list of users and an active user.